app/src/main/java/nodomain/freeyourgadget/gadgetbridge/impl/GBDevice.java

```diff
@@ -38,7 +40,10 @@ public GBDevice createFromParcel(Parcel source) {
     private String mHardwareVersion = null;
     private State mState = State.NOT_CONNECTED;
     private short mBatteryLevel = BATTERY_UNKNOWN;
-    private String mBatteryState;
+    private short mBatteryThresholdPercent = BATTERY_THRESHOLD_PERCENT;
+    //TODO: get rid of String mBatteryStatus in favor of Enum mBatteryState
+    private String mBatteryStatus;
```

Comment:
Well, they're still there, both the mBatteryStatus as well as the TODO, aren't they?